Movie Challenge/Movie Challenge/Controllers/Home/HomeViewController.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  HomeViewController.swift
+//  Movie Challenge
+//
+//  Created by Igor Gomes Arantes on 14/09/2018.
+//  Copyright © 2018 igor gomes arantes. All rights reserved.
+//
+
+import UIKit
+
+class HomeViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, UICollectionViewDataSource, UICollectionViewDelegate{
+    
+    @IBOutlet weak var suggestionTableView: UITableView!
+    @IBOutlet weak var bestMovieView: UIView!
+    @IBOutlet weak var bestMovieImageView: UIImageView!
```

Comment:
Call your IBOutlet just only of label/Button/Etc. All of them be Views, this is intrinsics. Use the Xcode function "rename" to do this.